identity-wallet/src/state/reducers/authorization.rs

```diff
@@ -211,6 +221,8 @@ pub async fn handle_siopv2_authorization_request(state: AppState, _action: Actio
         })
     };
 
+    persist_asset("issuer_0", &connection_id).ok();
+
```

Comment:
Consider moving this up on top of:
```rust
if result.is_none() {
        connections.push(Connection {
            id: connection_id.clone(),
            client_name,
            url: connection_url,
            verified: false,
            first_interacted: connection_time.clone(),
            last_interacted: connection_time,
        })
    };
```

So that ` id: connection_id.clone()` does not need to be cloned.